docs/DevelopmentGuide.md

```diff
@@ -126,3 +126,13 @@ typedef struct _ebpf_widget
 Above all, if a file happens to differ in style from these guidelines (e.g.,
 private members are named `m_member` rather than `_member`), the existing style
 in that file takes precedence.
+
+### Project files
+
+All project files should include the following line to enable automated code analysis.
+
+```
+<Import Project="$(SolutionDir)\Analyze.default.props" />
```

Comment:
```suggestion
<Import Project="$(SolutionDir)Analyze.default.props" />
```
Actually I don't think this should be needed at all, and all the vcxproj changes can be reverted.
You can just use a Directory.Build.props file as documented in https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019

Reply:
Thanks. I didn't know that. This is a much better option. :-)